hoprd/rest-api/src/tickets.rs

```diff
@@ -174,7 +174,7 @@ pub(super) async fn show_ticket_statistics(State(state): State<Arc<InternalState
         delete,
         path = const_format::formatcp!("{BASE_PATH}/tickets/statistics"),
         responses(
-            (status = 204, description = "Ticket statistics reset successfully."),
```

Comment:
I believe 204 was correct here, it should be changed in the function to return 204 instead of 200.